src/common/components/ManualContactPopup.js

```diff
@@ -135,8 +175,7 @@ const ManualContactPopup=({show,
                   onFocus={() => HandleFocus("last_name")}
                   HandelKeyPress={(event)=>{
                     if (event.which === 13 ) {
-                        handleBtnClick()
-                    }}}
+                    validateLastName(event)}}}
```

Comment:
Here also, enable the enter key event and don't validate the last name on key press